scripts/InOutActivityRatio.py

```diff
@@ -15,29 +15,14 @@ def main():
     ###EVERYTHING CURRENTLY MAPS TO MODE_OFOPERARION = 1
 
     # Parameters to print over
-    regions = functions.initializeRegions()
-    subregions = functions.initializeSubregionsAsList()
-    years = functions.initializeYears()
-
-    ###########################################
-    # Get Min and Rnw techs
-    ###########################################
-
-    #Read in master list of technologies
-    dfGeneration_raw = pd.read_csv('../dataSources/techList_AUTO_GENERATED.csv')
-
-    #Populate techs by type dictionary
-    dfGeneration = dfGeneration_raw.loc[dfGeneration_raw['GENERATION'] == 'RNW']
-    rnwTechs = dfGeneration['VALUE'].tolist()
-    dfGeneration = dfGeneration_raw.loc[dfGeneration_raw['GENERATION'] == 'MIN']
-    minTechs = dfGeneration['VALUE'].tolist()
-    dfGeneration = dfGeneration_raw.loc[dfGeneration_raw['GENERATION'] == 'PWR']
-    pwrTechs = dfGeneration['VALUE'].tolist()
+    regions = functions.openYaml().get('regions')
+    subregions = functions.openYaml().get('subregions_list')
+    years = functions.openYaml().get('years')
 
     ###########################################
     # Tech to Fuel Mapping
     ###########################################
-    techToFuel = {
+    TECH_TO_FUEL = {

```

Comment:
We should maybe think of a better way to represent this? Either maybe in the config file or via an input csv file? Not necessary for this PR tho

Reply:
Good point, I can definitely put this in the config file.